src/main/java/ia/gateways/FileAccessGateway.java

```diff
@@ -0,0 +1,5 @@
+package ia.gateways;
+
+public interface FileAccessGateway {
```

Comment:
I believe we have to merge Nicholas' latest PR before I can extend the necessary file access gateways for the use cases? @HarveyRDonnelly

Reply:
Yes but that may be added in a future PR.